gazelle/python/testdata/with_third_party_requirements_from_imports_legacy/gazelle_python.yaml

```diff
@@ -0,0 +1,1678 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+manifest:
+  modules_mapping:
+    cachetools: cachetools
+    cachetools.__init__: cachetools
+    cachetools.func: cachetools
+    cachetools.keys: cachetools
+    certifi: certifi
```

Comment:
This looks like copy paste from somewhere else. The codebase no longer generates these long files. Could you please fix this?

Reply:
According to https://github.com/bazelbuild/rules_python/blob/36ed9e029c275256b520b1b5d0d3329664441ae0/gazelle/python/testdata/with_third_party_requirements_from_imports/README.md. The gazelle_python.yaml is generated from https://github.com/sramirezmartin/gazelle-toy-example/blob/main/subdir1/requirements_lock.txt.

Do you think I should add this file to git?

Reply:
Just cleaning up the `gazelle_python.yaml` would be sufficient here. This is test data. Having it generated may be too much upkeep.

Reply:
test `with_third_party_requirements_from_imports_legacy` has been removed. It was meant to be a compatibility test for the original format. But the guys' manual testing has ensured its usability.

Reply:
Removing sounds good.